packages/connectivity/connectivity_macos/.gitignore

```diff
@@ -0,0 +1,2 @@
+.packages
+pubspec.lock
```

Comment:
nit: These are already in the root `.gitignore`, I don't think there's a reason to create a file with duplicate entries here.

Reply:
Done